docs/api.rst

```diff
@@ -49,11 +49,11 @@ A channel object consists of a
 
 - ``balance`` should be an integer of the amount of the ``token_address`` token we have available for transferring.
 
-- ``state`` should be the current state of the channel represented by a string.
+- ``state`` should be the current state of the channel represented by a constant.
   Possible value are:
-  - ``'open'``: The channel is open and tokens are tradeable
-  - ``'closed'``: The channel has been closed by a participant
-  - ``'settled'``: The channel has been closed by a participant and also settled.
+  - ``'channel.STATE_OPENED'``: The channel is open and tokens are tradeable
```

Comment:
Please do not change this here. The external user can't know of Python classes, he only needed to provide the constant strings as it was before. But make sure `open` -> `opened`